lib/msal-browser/src/cache/TokenCache.ts

```diff
@@ -268,7 +268,7 @@ export class TokenCache implements ITokenCache {
                 authority,
                 homeAccountId,
                 idTokenClaims,
-                this.cryptoObj,
+                this.cryptoObj.base64Decode,
```

Comment:
pass in the base64Decode function from ./encode/base64Decode.ts instead (for better minification)